data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/resistance/resistance_active_on_both.cfg

```diff
@@ -0,0 +1,30 @@
+#####
```

Comment:
We currently don't run wmllint on the test files, but running it manually reports `line 1: no textdomain string` for each file that lacks a `# wmllint: no translatables`.

Reply:
Updated.